src/JaxkDev/Vehicles/Handlers/EventHandler.php

```diff
@@ -209,4 +244,23 @@ public function onDataPacketEvent(DataPacketReceiveEvent $event): void{
 				break;
 		}
 	}
+
+	public function onInteract(PlayerInteractEvent $event){
+	    $player = $event->getPlayer();
+        $item = $event->getItem();
+
+        if($event->isCancelled()) return;
+        //$item = Item::get(Item::NETHER_STAR, 100, 1);
+       // $item = $netherStart->getNamedTagEntry("car");
+       /* $item = Item::get(Item::TRIPWIRE_HOOK, 0, 1);
+        $item->setCustomName(TextFormat::BOLD.TextFormat::RED."BasicCar");
+        $item->setNamedTagEntry(new IntTag("car", 1));
+        $item->getNamedTagEntry("car");*/
+        if($item->getNamedTagEntry("car") === null){
+            return;
+        }
+        $this->plugin->factory->spawnVehicle($this->plugin->factory->getVehicleData("BasicCar"), $player->getLevel(), $player->asVector3());
```

Comment:
Extremely limiting, all custom vehicles will be lost via this method and "BasicCar" may not exist.